events/2020-10-10.interspeciesio-ml-workshop.md

```diff
@@ -1,25 +1,109 @@
+*   [Efficient Trainable Front-Ends for Neural Speech Enhancement](https://arxiv.org/abs/2002.09286)
+*   [Finding Strength in Weakness: Learning to Separate Sounds with Weak Supervision](https://arxiv.org/abs/1911.02182)
+*   [Monaural Source Separation Based on Sequentially Trained LSTMs in Real Room Environments](https://ieeexplore.ieee.org/document/8902640)
+*   [The Phasebook: Building Complex Masks via Discrete Representations for Source Separation](https://waseda.pure.elsevier.com/en/publications/the-phasebook-building-complex-masks-via-discrete-representations)
+*   [Universal Sound Separation](https://arxiv.org/abs/1905.03330)
+*   [Speaker-independent Speech Separation with Deep Attractor Network](https://arxiv.org/abs/1707.03634)
+*   [Deep Clustering and Conventional Networks for Music Separation: Stronger Together](https://arxiv.org/abs/1611.06265)
+*   [Phase-Sensitive and Recognition-Boosted Speech Separation Using Deep Recurrent Neural Networks](https://ieeexplore.ieee.org/document/7178061)
+*   [Joint Optimization of Masks and Deep Recurrent Neural Networks for Monaural Source Separation](https://arxiv.org/abs/1502.04149)
+*   [Conv-TasNet: Surpassing Ideal Time-Frequency Magnitude Masking for Speech Separation](https://arxiv.org/abs/1809.07454)
+*   [Deep Learning Based Phase Reconstruction for Speaker Separation: A Trigonometric Perspective](https://arxiv.org/abs/1811.09010)
+*   [Divide and Conquer: A Deep CASA Approach to Talker-independent Monaural Speaker Separation](https://arxiv.org/abs/1904.11148)
+*   [Dual-Path RNN: Efficient Long Sequence Modeling for Time-Domain Single-Channel Speech Separation](https://arxiv.org/abs/1910.06379)
+*   [A Comprehensive Study of Speech Separation: Spectrogram vs Waveform Separation](https://arxiv.org/abs/1905.07497)
+*   [FurcaNeXt: End-to-End Monaural Speech Separation with Dynamic Gated Dilated Temporal Convolutional Networks](https://arxiv.org/abs/1902.04891)
+*   [Improving Universal Sound Separation Using Sound Classification](https://arxiv.org/abs/1911.07951)
+*   [SpEx: Multi-Scale Time Domain Speaker Extraction Network](https://arxiv.org/abs/2004.08326)
+*   [Meta-Learning Extractors for Music Source Separation](https://arxiv.org/abs/2002.07016)
+*   [Unsupervised Learning of Semantic Audio Representations](https://arxiv.org/abs/1711.02209)
+*   [Sudo rm -rf: Efficient Networks for Universal Audio Source Separation](https://arxiv.org/abs/2007.06833)
+*   [Unsupervised Sound Separation Using Mixtures of Mixtures](https://arxiv.org/abs/2006.12701)
+*   [Listen to What You Want: Neural Network-based Universal Sound Selector](https://arxiv.org/abs/2006.05712)
+*   [Speech Separation Based on Multi-Stage Elaborated Dual-Path Deep BiLSTM with Auxiliary Identity Loss](https://arxiv.org/abs/2008.03149)
+*   [Identify Speakers in Cocktail Parties with End-to-End Attention](https://arxiv.org/abs/2005.11408)
```

Comment:
This is frickin epic. @pcbermant, anything to add here?

Reply:
Those are all directly from @pcbermant 's readme in the Cocktail Party bookshelf, so I would assume he's good with it for us to push live?

Reply:
Is it worth linking directly to the bookshelf? There are a ton of other resources in there, but these ones seemed to be some of the most noteworthy.

Reply:
This is so good!